spec/reek/spec/should_reek_of_spec.rb

```diff
@@ -6,8 +6,8 @@
   describe 'smell type selection' do
     let(:ruby) { 'def double_thing() @other.thing.foo + @other.thing.foo end' }
 
-    it 'reports duplicate calls by smell category' do
-      expect(ruby).to reek_of(:Duplication)
+    it 'does not report duplicate calls by smell category' do
```

Comment:
Wouldn't it make more sense to remove this spec completely?
This will make no sense to the innocent reader in 4 weeks who doesn't know that smell_category never existed.